main.nf

```diff
@@ -81,83 +123,360 @@ if( !(workflow.runName ==~ /[a-z]+_[a-z]+/) ){
   custom_runName = workflow.runName
 }
 
+/*
+ * short read preprocessing options
+ */
+params.adapter_forward = "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA"
+params.adapter_reverse = "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT"
+params.mean_quality = 15
+params.trimming_quality = 15
+params.keep_phix = false
+// params.phix_reference = "ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/viral/Enterobacteria_phage_phiX174_sensu_lato/all_assembly_versions/GCA_002596845.1_ASM259684v1/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
+params.phix_reference = "$baseDir/assets/data/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
+
+/*
+ * binning options
+ */
+params.skip_binning = false
+params.skip_busco = false
+params.min_contig_size = 1500
+
+/*
+ * assembly options
+ */
+params.skip_spades = false
+params.skip_spadeshybrid = false
+params.skip_megahit = false
+params.skip_quast = false
+
+/*
+ * taxonomy options
+ */
+params.centrifuge_db = false
+params.kraken2_db = false
+params.skip_krona = false
+params.cat_db = false
+
+/*
+ * long read preprocessing options
+ */
+params.skip_adapter_trimming = false
+params.keep_lambda = false
+params.longreads_min_length = 1000
+params.longreads_keep_percent = 90
+params.longreads_length_weight = 10
+// params.lambda_reference = "ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/viral/Escherichia_virus_Lambda/all_assembly_versions/GCA_000840245.1_ViralProj14204/GCA_000840245.1_ViralProj14204_genomic.fna.gz"
+params.lambda_reference = "$baseDir/assets/data/GCA_000840245.1_ViralProj14204_genomic.fna.gz"
+
+// Stage config files
+ch_multiqc_config = Channel.fromPath(params.multiqc_config)
+ch_output_docs = Channel.fromPath("$baseDir/docs/output.md")
+
 /*
  * Create a channel for input read files
  */
-Channel
-    .fromFilePairs( params.reads, size: params.singleEnd ? 1 : 2 )
-    .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}\nNB: Path needs to be enclosed in quotes!\nNB: Path requires at least one * wildcard!\nIf this is single-end data, please specify --singleEnd on the command line." }
-    .into { read_files_fastqc }
+if(!params.skip_busco){
+    Channel
+        .fromPath( "${params.busco_reference}", checkIfExists: true )
+        .set { file_busco_db }
+} else {
+    file_busco_db = Channel.from()
```

Comment:
```suggestion
    file_busco_db = Channel.from(false)
```
I forget why the `false` is needed, but I remember hitting problems without it before I think..